app/views/planning_applications/assessment/local_policies/show.html.erb

```diff
@@ -14,7 +14,13 @@
       locals: {heading: "Assess against policies and guidance"}
     ) %>
 
-<%= govuk_link_to "Check your local policies and guidance in a new tab", I18n.t("council_documents.#{@planning_application.local_authority.subdomain}.planning_policy_and_guidance"), target: "_blank", class: "govuk-body" %>
+<p class="govuk-body">
+  <%= govuk_link_to "Check your local policies and guidance in a new tab", I18n.t("council_documents.#{@planning_application.local_authority.subdomain}.planning_policy_and_guidance"), target: "_blank" %>
+</p>
+
+<p class="govuk-body">
+  <%= govuk_link_to "View draft decision notice (in a new tab)", decision_notice_planning_application_path(@planning_application), target: "_blank" %>
+</p>
```

Comment:
Is this better as a partial if we need to reuse it?

Reply:
Couldn't hurt!